src/chromyRunner.test.js

```diff
@@ -69,7 +69,6 @@ describe('ChromyRunner', () => {
       expect(loggerCalls[0]).toEqual('goto -> www.example.com');
       expect(loggerCalls[1]).toEqual('Capturing screenshot of whole DOM');
       expect(loggerCalls[2]).toEqual('screenshot saved in -> ./differencify_report/default.png');
-      expect(loggerCalls[4]).toEqual('Saving the diff image to disk');
```

Comment:
testing logs is not a good way of verifying behaviour, so I didn't update it. `src/chromyRunner.test.js` needs a big refactor IMO